docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -433,9 +462,48 @@ on two data sets and different hardware configurations. Your data set has an
 impact on the model performance. Run tests on your own data to have a more 
 realistic view on the model performance for your use case.
 
+[discrete]
+==== Summary
+
+Overall the platform specific V2 model ingested at a max rate of 25.90 docs/s, 
+compared with the ELSER V1 max rate of 13.65 docs/s from the ELSER V1 benchamrk, 
+resulting in a 90% increase in throughput.
+
+Furthermore, a significant performance improvement is found in ELSER V1 on 8.11, 
+when compared to 8.9, with maximum throughputs of 16.13 and 13.65 docs/s 
+respectively.
+
+The performance of virtual cores (that is, when the number of allocations is 
+greater than half of the vCPUs) has increased. Previously, the increase in 
+performance between 8 and 16 allocations was around 7%. It has increased to 17% 
```

Comment:
Since we are mentioning the number of allocations, we should specify that these tests were performed on a 16vCPU machine.